src/expandable-section/styles.scss

```diff
@@ -87,55 +86,57 @@ $icon-total-space-medium: calc(#{$icon-width-medium} + #{$icon-margin-left} + #{
   }
 
   &-default {
-    padding-block: awsui.$space-scaled-xxs;
-    padding-inline-end: awsui.$space-xxs;
+    padding-top: awsui.$space-scaled-xxs;
+    padding-bottom: awsui.$space-scaled-xxs;
+    padding-right: awsui.$space-xxs;
     &.header-deprecated {
-      padding-inline-start: awsui.$space-xxs;
+      padding-left: awsui.$space-xxs;
     }
     &:not(.header-deprecated) {
-      padding-inline-start: calc(#{awsui.$space-xxs} + #{$icon-total-space-normal});
+      padding-left: calc(#{awsui.$space-xxs} + #{$icon-total-space-normal});
     }
   }
 
   &-footer {
-    padding-block: awsui.$space-scaled-xxs;
+    padding-top: awsui.$space-scaled-xxs;
+    padding-bottom: awsui.$space-scaled-xxs;
   }
 
   &-footer,
   &-compact {
-    padding-inline-end: 0;
+    padding-right: 0;
     &.header-deprecated {
-      padding-inline-start: 0;
+      padding-left: 0;
     }
     &:not(.header-deprecated) {
-      padding-inline-start: $icon-total-space-normal;
+      padding-left: $icon-total-space-normal;
     }
   }
 
```

Comment:
There was a merge conflict with this PR #1644 which was merged in between.

I think I resolved all conflicts correctly, just FYI